ggu/main.c

```diff
@@ -880,13 +880,15 @@ run_ggu(struct net_config *net_conf,
  * Return an error if trying to allocate the second instance.
  */
 struct ggu_config *
-alloc_ggu_conf(void)
+alloc_ggu_conf(unsigned int lcore)
 {
 	static rte_atomic16_t num_ggu_conf_alloc = RTE_ATOMIC16_INIT(0);
 
 	if (rte_atomic16_test_and_set(&num_ggu_conf_alloc) == 1) {
-		ggu_conf = rte_calloc("ggu_config", 1,
-			sizeof(struct ggu_config), 0);
+		ggu_conf = rte_calloc_socket("ggu_config", 1,
```

Comment:
Wow, this is a rather old bug: we are not testing the case that `rte_calloc_socket()` returns `NULL`.